templates/cloud/plans-and-pricing.html

```diff
@@ -154,7 +178,13 @@
                 </tr>
                 <tr>
                   <th rowspan="1" scope="row"><a href="/support">Landscape</a></th>
-                  <td>Sold as part of Ubuntu Advantage</td>
+                  <td>
+                      Free up to ten machines <br />
+                      or <br />
+                      $0.01 per machine hour (SaaS) <br />
+                      or <br />
+                      sold as part of Ubuntu Advantage (SaaS or on-premises)
```

Comment:
Spotted this, I thought the copy doc was wrong. @Greggless What you think? Capital `S` even though it's a continuation of the sentence?